pages/en/lb4/Getting-started.md

```diff
@@ -5,35 +5,202 @@ keywords: LoopBack 4.0, LoopBack 4
+
+const app = new Application();
+app.bind('message').to('Hello worldA');
+app.get('message').then(value => {
+  console.log(value);
+});
+```
+
+### Run the project
+
+To run your project, enter this command:
+
+```shell
+node index.js
+```
+
+You should see "Hello world!" written to the console.
+
+## Try a TypeScript project
+
+If you're coding in TypeScript, follow these steps:
+
+1. Create `tsconfig.json` file.
+2. Create `index.ts` file.
+1. Run the project.
+
+### Create `tsconfig.json` file
+
+A `tsconfig.json` file in a directory indicates that it contains a TypeScript project. The `tsconfig.json` file specifies the root files and the TypeScript compiler options. For more information on `tsconfig.json`, see [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).
+
+Create a new `tsconfig.json` file by entering the following command:
+
+```
+$ tsc --init
+```
+
+Now edit this file, because you need to change some of the default values.
+
+{% include important.html content= "You must set the `'compilerOptions.target'` property in  your project's `tsconfig.json` file based on the version of Node you are using.
+" %}
+
+**With Node 6.x**:
+
+```js
+{
+  "compilerOptions": {
+   ...
+    "target": "es6",
+    ...
+  }
+}
+```
+
+**With Node 8.x**:
+
+```js
+{
+  "compilerOptions": {
+    ...
+    "target": "es2017",
+    ...
+  }
+}
+```
+
+### Create `index.ts`
+
+Create a file named `index.ts` and copy the following code into it:
+
+```ts
+import {Application} from '@loopback/core';
+
+const app = new Application();
+app.bind('message').to('Hello world!');
+app.get('message').then(value => {
+  console.log(value);
+});
+```
+
+### Run the project
+
+Then run `index.ts` by entering this command:
+
+```
+ts-node index.ts
+```
+
+You should see "world" written to the console.
+
+## Try a more complex TypeScript project
+
+This example builds on the basic "Hello world" example with an `Application` and a `RestServer` that responds to all HTTP requests with the text "Hello World".
+
+{% include note.html content= "This section assumes you've already created the
+[basic TypeScript project](#try-a-typescript-project).
 " %}
 
-## Installation
+### Install LoopBack REST package
 
-Make sure you have Node.js 8.0.0 or higher installed.
+Install the LoopBack REST package with npm by entering this command:
 
-## A simple example
+```js
+npm i -s @loopback/rest
+```
 
-This example creates an `Application` and a `RestServer` that responds to all HTTP requests with the text "Hello World".
+This adds the `@types/node` package to the `dependencies` property of `package.json`.
 
-### Building a LoopBack 4 application
-First create a new folder.  Then, create three files within this folder: `index.ts`, `tsconfig.json` and `package.json`.  
+
+### Install Node type definitions package
+
+Install the `@types/node` package as a development dependency by entering this command:
+
+```js
+npm i --save-dev @types/node
```

Comment:
If we are using `ts-node` to run the app, then `@types/node` is a regular runtime (production) dependency.

It would be a dev-dependency if we were using `tsc` to transpile from TS to JS at dev-time and then run the transpiled javascript in production.